frontend/src/components/App.js

```diff
@@ -104,7 +104,7 @@ export default class App extends Component {
     let destCourseIndex = result.destination.index;
     let sourceCourseIndex = result.source.index;
 
-    if (result.source.droppableId === 'search-results') {
+    if (result.source.droppableId.includes('search-result-droppable')) {
```

Comment:
Since we have a different Droppable for each SearchCourseCard, we need to change this logic here